wallet_ui/components/panels/CreateCanister.tsx

```diff
@@ -133,11 +131,9 @@ export function CreateCanisterDialog(props: {
   function create() {
     setLoading(true);
 
-    let additional = moreControllers.filter(ea => ea.length !== 0);
-    let allControllers = [controller, ...additional];
-    let result = allControllers.map(ea => Principal.fromText(ea));
-    const args = {result, cycles};
-
+    let controllers = controllersState.filter(ea => ea.length !== 0).map(ea => Principal.fromText(ea));
```

Comment:
As a code linting rule, use `const` instead of `let` when the variable will not be changed below
```suggestion
    const controllers = controllersState.filter(ea => ea.length !== 0).map(ea => Principal.fromText(ea));
```